interface/Yami.h.in

```diff
@@ -19,10 +19,17 @@
 
 #include <YamiVersion.h>
 
+#if (@__YAMI_HAS_DECODER__@)
```

Comment:
why we need this, include header do not enlarger the binary size

Reply:
agree with you @xuguangxin.  
But if no decoder is compiled, the sub-directory "decoder" will not be included into the compilation, and the base class files(vaapidecpicture.cpp, vaapidecsurfacepool.cpp...) are not be compiled,  the corresponding head files(VideoDecoderInterface.h, VideoDecoderHost.h) also are not installed, so Yami.h can not include decoder's head file(VideoDecoderHost.h).

It's okay just to install decoder's head files, and not compile the cpp files, but it will bring some confusing, and we have to change the Makefile.am too.